//package com.example.learnjavacore.Exception;
//import java.lang.Exception;
//public class TryCatch {
//    /*
//        - Khối try trong Java được sử dụng để bao quanh code mà có thể ném một Exception.
//            + Nó được sử dụng bên trong phương thức.
//            + Khối try phải ược theo sau bởi khối catch hoặc khối finally.
//
//     */
//    public static void main(String[] args) {
//        tryCatch();
//        tryFinally();
//    }
//    public static void tryCatch()
//    {
//        try {
//            // code mà có thể ném exception
//        }catch (Exception exception)
//        {
//
//        }
//    }
//    public static void tryFinally()
//    {
//        try {
//            // code mà có thể ném exception
//        }finally {
//
//        }
//    }
//}
